ci/elixir.yml

```diff
@@ -12,10 +12,24 @@ jobs:
 
     steps:
     - uses: actions/checkout@v1
+    - name: Cache Mix Dependencies
+      uses: actions/cache@v1
+      with:
+        path: deps
+        key: ${{ runner.OS }}-mix-${{ hashFiles('**/mix.lock') }}

```

Comment:
https://github.com/actions/cache/blob/master/examples.md#elixir---mix has an example that is slightly different. I guess the difference is whether it only uses the top level mix.lock or not. No idea which is preferable.

Reply:
I think it's fine to play it safe here and use `**/mix.lock`.